src/System.Management.Automation/commands/utility/FormatAndOutput/common/FormatXMLWriter.cs

```diff
@@ -37,7 +38,7 @@ internal class FormatXmlWriter
             StreamWriter streamWriter;
             FileStream fileStream;
             FileInfo fileInfo;
-            PathUtils.MasterStreamOpen(cmdlet, filepath, "ascii", true, false, force, noclobber,
+            PathUtils.MasterStreamOpen(cmdlet, filepath, PowerShellEncoding.GetEncoding(FileEncoding.Ascii), true, false, force, noclobber,
```

Comment:
yah - I was worried about removing the overload as it was a public interface and thought to do less harm, just in case someone else was using it somehow.